bin/create.js

```diff
@@ -364,6 +449,22 @@ function parseDate (argv, field) {
     argv[field] = Math.floor(date.valueOf() / 1000);
 }
 
+function getSecret() {
+    try {
+        return (process.env.WEBTASK_JWT_SECRET || Crypto.randomBytes(128).toString('base64'));
+    } catch(e) {
+        console.log('Couldn\'t generate random secret: ' + e.message);
```

Comment:
Should this just throw? In any way it probably shouldn't silently return undefined.